testing/environments/ports.yml

```diff
@@ -11,7 +11,3 @@ logstash:
 kibana:
   ports:
     - "5601:5601"
-
-beat:
```

Comment:
@monicasarbu No. It is on purpose not in the base.yml file as if it is in the base file it is fetched and built for each CI build on travis and jenkins which makes the builds a lot slower.